### PR TITLE
enhance chatbot code

### PR DESCRIPTION
**Pull request by lkk12014402**
## Type of Change

API not changed

## Description

1.	Add parameter “--task” = [chat, instruction, summarization] to indicate different task dataset format
2.	Describe chatbot/instruction/summarization dataset format in README
3.	Enhance generate script for chabot/summarization

### CODE REVIEW

File: workflows/chatbot/fine_tuning/README.md [hunk 1]
```diff
@@ -38,13 +38,92 @@ We select 4 kind of datasets to conduct the finetuning process for different tas
...
+        ### Instruction:
+        {instruction}
+    
+        ### Input:
+        {input}
+
+        ### Response:
+
+        ```
+        - for examples with a non-empty input field:
```
Comment (hshen14) [inline workflows/chatbot/fine_tuning/README.md:64]: An example without input field:

File: workflows/chatbot/fine_tuning/README.md [hunk 2]
```diff
@@ -38,13 +38,92 @@ We select 4 kind of datasets to conduct the finetuning process for different tas
...
+    ```python
+    {
+        "instruction": "Give three tips for staying healthy.",
+        "input": "",
+        "output": "1.Eat a balanced diet and make sure to include plenty of fruits and vegetables. \n2. Exercise regularly to keep your body active and strong. \n3. Get enough sleep and maintain a consistent sleep schedule."
+    }
+    ```
+
+    - **Instruction template:**
+        - for examples with a non-empty input field:
```
Comment (hshen14) [inline workflows/chatbot/fine_tuning/README.md:51]: An example with an input field:

File: workflows/chatbot/fine_tuning/README.md (hunk 1 above)
Comment (lkk12014402) [reply workflows/chatbot/fine_tuning/README.md]: fix

File: workflows/chatbot/fine_tuning/README.md (hunk 2 above)
Comment (lkk12014402) [reply workflows/chatbot/fine_tuning/README.md:58]: fix

**Review by hshen14 — APPROVED** [review]

**Review by sywangyi — APPROVED** [review]

**Pull request merged**